src/services/pcn-firewall/datamodel/firewall.yang

```diff
@@ -92,8 +92,8 @@ module firewall {
     leaf action {
       type action;
       polycube-base:init-only-config;
-      description "Action if the rule matches. Default is DROP.";
-      polycube-base:cli-example "DROP, FORWARD, LOG";
+      description "Action if the rule matches. Default is ACCEPT.";
```

Comment:
Default action should be DROP not ACCEPT

Reply:
This was changed recently because of a logistical problem.
Let's assume you deploy a firewall on a remote machine, and the default action is DROP. What happens, is that as soon as you start the firewall, you loose the connection to the remote machine.
So, to avoid this problem (very important in real deployments), we changed the default action to ACCEPT. Obviously, the user can change it to DROP upon its preferences, hopefully after having properly configured the ruleset.

Reply:
@frisso A better practice without changing the default policy is 
1) create firewall cube 
2) configure the firewall with rule accepting the management traffic 
3) attach the firewall cube to the interface

Reply:
If we want to use iptables as an example of firewall behaviour I think that ACCEPT is the default action (for example after a fresh Ubuntu server installation)

Reply:
@goldenrye What you say is definitely safer, but in our experience this would trigger a lot of problems, because many people would  first start and attach the firewall, then configure it. So, they would be cut out of their environment because the default 'deny' rule. I would prefer to stay with a default "accept" and improve the documentation in order to warn about this problem. Btw, we can explicitly state that the preferred configuration choice, if you want to have a default 'deny' rule, is to configure the rule, then attach the firewall to the ports.